lib/ui/profile/components/profile_view.dart

```diff
@@ -61,8 +62,35 @@ class ProfileView extends StatelessWidget {
                             onImageRemoved: () {
                               context.read<ProfileBloc>().add(const ProfileEvent.onRemoveImageTapped());
                             },
-                            onImageSelected: (image) async =>
-                                context.read<ProfileBloc>().add(ProfileEvent.setAvatarImage(image)),
+                            onImageSelected: (image) async {
+                              final CroppedFile? croppedFile = await ImageCropper().cropImage(
+                                sourcePath: image.path,
+                                aspectRatioPresets: [
```

Comment:
It would make sense to set the aspect preset to square only?! Since the profile image is in a round frame all the time.